eox_core/social_tpa_backends.py

```diff
@@ -51,3 +51,13 @@ def oidc_config(self):
                 self.OIDC_ENDPOINT + '/.well-known/openid-configuration'
             )
         return self.configuration
+
+    def extra_data(self, user, uid, response, details=None, *args, **kwargs):  # pylint: disable=keyword-arg-before-vararg
+        """
+        Override method extra_data returned by the provider including user
+        information.
+        """
+        data = super(ConfigurableOpenIdConnectAuth, self).extra_data(user, uid, response, details, *args, **kwargs)
```

Comment:
```suggestion
        data = super().extra_data(user, uid, response, details, *args, **kwargs)
```
Since we dropped support for Python 2.